src/components/Explore/MultipleLocationsChart.tsx

```diff
@@ -133,7 +133,9 @@ const MultipleLocationsChart: React.FC<{
   barOpacity,
   isMobileXs = false,
 }) => {
-  const seriesList = sortSeriesByLast(unsortedSeriesList);
+  const seriesList = sortSeriesByLast(unsortedSeriesList).filter(
+    series => series.data.length > 0,
+  );
```

Comment:
This filter ensures we don't include empty series and show a label in the upper left of the chart.